graylog2-server/src/test/java/org/graylog/plugins/views/search/rest/QualifyingViewsResourceTest.java

```diff
@@ -16,98 +16,98 @@
  */
 package org.graylog.plugins.views.search.rest;
 
-import com.google.common.collect.ImmutableList;
 import org.graylog.plugins.views.search.permissions.SearchUser;
 import org.graylog.plugins.views.search.views.QualifyingViewsService;
 import org.graylog.plugins.views.search.views.ViewDTO;
 import org.graylog.plugins.views.search.views.ViewParameterSummaryDTO;
-import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
```

Comment:
As you have made a great effort to clean up many tests, it is very likely that you can safely migrate them to JUnit5 as well, replacing this import with "org.junit.jupiter.api.Test" ! :)
It may be not the only test in this PR that can be safely migrated.